packages/boba/subgraphs/anchorage-dgf-sepolia/subgraph.yaml

```diff
@@ -75,40 +79,44 @@ dataSources:
           handler: handleOwnershipTransferred
       file: ./src/dispute-game-factory.ts
 templates:
-  - name: PermissionedDisputeGame
+  - name: FaultDisputeGame
     kind: ethereum/contract
     network: sepolia
     source:
-      abi: PermissionedDisputeGame
+      abi: FaultDisputeGame
     mapping:
       kind: ethereum/events
       apiVersion: 0.0.7
       language: wasm/assemblyscript
-      file: ./src/permissioned-dispute-game.ts
+      file: ./src/dispute-game.ts
       entities:
         - Resolved
       abis:
+        - name: FaultDisputeGame
+          file: ./abis/FaultDisputeGame.json
         - name: PermissionedDisputeGame
           file: ./abis/PermissionedDisputeGame.json
       eventHandlers:
-        - event: Resolved(indexed uint8)
-          handler: handleResolved
+        - event: Resolved(indexed uint8,indexed bytes32,indexed uint256)
+          handler: handleResolvedLegacy
 
-  - name: FaultDisputeGame
+  - name: PermissionedDisputeGame
     kind: ethereum/contract
     network: sepolia
     source:
-      abi: FaultDisputeGame
+      abi: PermissionedDisputeGame
     mapping:
       kind: ethereum/events
       apiVersion: 0.0.7
       language: wasm/assemblyscript
-      file: ./src/fault-dispute-game.ts
+      file: ./src/dispute-game.ts
       entities:
         - Resolved
       abis:
         - name: FaultDisputeGame
           file: ./abis/FaultDisputeGame.json
+        - name: PermissionedDisputeGame
+          file: ./abis/PermissionedDisputeGame.json
       eventHandlers:
-        - event: Resolved(indexed uint8,indexed bytes32,indexed uint256)
-          handler: handleResolved
+        - event: Resolved(indexed uint8)
```

Comment:
This doesn't seem quite right.

Per my reading, only templates from `PermissionedDisputeGame` will capture the `Resolved(indexed uint8)` event.

We need this event for both the FaultDisputeGame, and the PermissionedDisputeGame.  I think we need to copy this event handler up into the previous template, and remote references to the fault dispute game in this template?

Reply:
could you elaborate why you think this doesn't work?

The template does listen on both eventHandlers and ABIs.

Reply:
ah got you, yeah probably not. As you indicated likely not necessary. Removed and redeployed to `v1.0.11`.